pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -46,6 +46,9 @@ type VPCNetworkConfigurationSpec struct {
 	// +kubebuilder:validation:MaxLength=8
 	// +optional
 	ShortID string `json:"shortID,omitempty"`
+	// Policy path of the NSX VPC the Namespace associated with.
```

Comment:
Could we change to "Policy path" to "NSX path"? Users of NSX operator CRs are confused for "policy path".

Reply:
updated.